cookbook/signals.py

```diff
@@ -161,3 +163,9 @@ def clear_unit_cache(sender, instance=None, created=False, **kwargs):
 def clear_property_type_cache(sender, instance=None, created=False, **kwargs):
     if instance:
         caches['default'].delete(CacheHelper(instance.space).PROPERTY_TYPE_CACHE_KEY)
+
+
+if ENABLE_EXTERNAL_CONNECTORS:
+    handler = ConnectorManager()
+    post_save.connect(handler, dispatch_uid="connector_manager")
```

Comment:
does this add the post_save to all models? should we limit this to only signal on save events of ShoppingListEntry objects?

Reply:
I understand that it is more error prone but I am unsure about performance. Tandoor useses a ton of models so many many many changes are triggered on a lot of operations. 

Signals are far from free in regards of performance so their use should be limited as good as possible so I suggest looking into if a more limited signal could be added.